test/suite/__snapshots__/extension.test.js.snap

```diff
@@ -23,6 +23,17 @@ export { options };
 "
 `;
 
+exports[`Extension Test Suite setting "eslintConfigPath" uses the target .eslintrc 1`] = `
+"const options = {
+    first: "hello, world",
+    second: true,
+    internationalizing: true,
```

Comment:
it was expected `internationalizing: true`(no trailing comma) but it doesn't work. @idahogurl Do you have any clues?